microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md

```diff
@@ -18,17 +18,17 @@ Generally, extensions are distributed through the Microsoft Edge Add-ons website
 Extensions that aren't loaded from the Edge Add-ons store are referred to as externally installed extensions. The following are alternate methods of distributing externally installed extensions:
 
 *  Use the Windows registry (Windows only).
-*  Use a preferences JSON file (macOS and Linux).
+*  Use a preferences JSON file (macOS and Linux). The preferences JSON file would be your **Microsoft Edge extension's CRX ID**.json. The CRX ID is a unique 32 character code which consists of alphabets present at the end of your Microsoft Edge extension's URL. 
```

Comment:
```suggestion
*  Use a preferences JSON file (macOS and Linux). If you use this approach, the name of the preferences JSON file is your Microsoft Edge extension's CRX ID, followed by a `.json` extension. The CRX ID is a unique 32-character code which is the letters that are present at the end of your extension's URL.
```
Engineering, please review this suggested change.

Reply:
The present comment thread is Outdated.  Latest:
https://github.com/MicrosoftDocs/edge-developer/pull/2272/files

Reply:
The article has been restructured to avoid premature detail.  Engr approved change.